cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BaseSubscriptionDTO.java

```diff
@@ -336,11 +354,19 @@ public void setBandwidth( BigDecimal bandwidth )
         this.bandwidth = bandwidth;
     }
 
+    /**
+     * @deprecated use {@link #getBandwidth()}
+     */
+    @Deprecated
     public BigInteger getBandwidthInGB()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)